### PR TITLE
[13.0][MIG] hr_employee_relative

### PR DESCRIPTION
**Pull request by Saran440**
test script depend on https://github.com/OCA/hr/pull/806

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-known_third_party=dateutil,openupgradelib,setuptools
```
Comment (alexey-pelykh) [inline .isort.cfg]: This change I guess is not needed?

**Review by alexey-pelykh — APPROVED** [review]

File: .isort.cfg (hunk 1 above)
Comment (Saran440) [reply .isort.cfg]: @alexey-pelykh Fixed, this is auto generate from pre-commit

File: hr_employee_relative/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Employee Relatives",
+    "version": "13.0.1.0.0",
```
Comment (alexey-pelykh) [inline hr_employee_relative/__manifest__.py:6]: Please split the isort and actual migration in two commits as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch

Comment (Saran440) [reply hr_employee_relative/__manifest__.py:6]: it's my mistake. I think change version before pre-commit. thank you for your review :)

**Comment by Saran440** [comment]
testscript depend on https://github.com/OCA/hr/pull/806

**Comment by alexey-pelykh** [comment]
@Saran440 how this depends on #806 ?

**Comment by Saran440** [comment]
@alexey-pelykh it's not depend but codecov compare test script with hr_employee_firstname. I don't know why.

**Comment by alexey-pelykh** [comment]
I think that commit history can be cleanup by merging as "fixup" the translation and bot commits to previous significant one, e.g.:
image
can be made a single commit and these:
image

Yet otherwise I guess we can merge

**Review by jarroyomorales — APPROVED** [review]
Tested on runbot, LGTM :+1:

**Comment by alexey-pelykh** [comment]
@Saran440 please squash commits as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0 and let's merge:

> Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot.

**Comment by Saran440** [comment]
@alexey-pelykh Sorry for late. Fixed. Thank you for reference :)

**Pull request merged**